### ISSUE TITLE
prescript actions don't seem to be working as documented

### ISSUE THREAD

**Issue by kcgthb** [xcat2/xcat-core#5676]
Hi,

I have a prescripts table that looks like this:
```
#node,begin,end,comments,disable
"cn","all_pre.sh|osimage:osimage_pre.sh",,,
```

A compute node (`cn`) that uses `sn` as a service node:
```
# lsdef -c cn -i servicenode,prescripts-begin
cn: prescripts-begin=all_pre.sh|osimage:osimage_pre.sh
cn: servicenode=sn
```

According to the `prescripts` [documentation](https://xcat-docs.readthedocs.io/en/stable/guides/admin-guides/references/man5/prescripts.5.html), I'm expecting to see:
* `nodeset cn osimage` to execute both `all_pre.sh` and `osimage_pre.sh` scripts
* `nodeset cn boot` to execute `all_pre.sh` only.

The observed behavior is:
* `nodeset cn osimage` only executes `osimage_pre.sh`
```
# nodeset cn osimage
cn: install centos7.5-x86_64-cpu
sn: Running begin script osimage_pre.sh for nodes cn.

cn: install centos7.5-x86_64-cpu
cn: install centos7.5-x86_64-cpu
```
* `nodeset cn boot` doesn't execute *ANY* prescript:
```
# nodeset cn boot
cn: boot
cn: boot
cn: boot
```

Could you please advise?

**Comment by robin2008** [comment]
This is fixed in #5682, @kcgthb  would it cover your case?

**Comment by kcgthb** [comment]
@robin2008 I believe so, thank you!
